Shared/Models/Eatery.swift

```diff
@@ -93,6 +93,10 @@ protocol Eatery {
 
     var paymentMethods: [PaymentMethod] { get }
 
+    var latitude: CLLocationDegrees { get }
+
+    var longitude: CLLocationDegrees { get }
+
     var location: CLLocation { get }
 
     var phone: String { get }
```

Comment:
I feel like all this spacing is unnecessary.

Reply:
This is to conform with the existing spacing in the Eatery file (but yes it is unnecessary). 

The context here is that some getters had documentation, so to keep formatting consistent between getters with documentation and getters without, there was a newline between each variable declaration.